### PR TITLE
Fix stdin test

### PR DESCRIPTION
**Pull request by 5bentz**
This PR includes a fix of a typo in the code.

When editing, I noticed I could improve the code, thus the rewrite. Using run() is the recommended approach according to Python.org. **It requires Python >= 3.5 (released in November 2015).**

I have also renamed the pref variables to uppercase since they are constants.
This change highlights the difference between the pref variables & 
the other variables in the code.

### CODE REVIEW

File: src/passff.py [hunk 1]
```diff
@@ -14,14 +14,14 @@
 ################################################################################
 # Default command for MacOS:
 #command = "/usr/local/bin/pass"
```
Comment (tuxor1337) [inline src/passff.py]: Please uppercase `command -> COMMAND` also in this line for Mac users.

**Comment by 5bentz** [comment]
Force-update done.
Mac users are happy again :+1:

**Comment by tuxor1337** [comment]
This is not urgent, right? Because it's working for me.

**Comment by 5bentz** [comment]
Not urgent at all. I am sorry for the misunderstanding.

If I'm correct, the code would *always* create a pipe for stdin, even when the code was not giving any input to `pass`.
All in all, it must have been harmless. This fix is like an excuse to DRY the code :smiley: 

Thanks for the merge :+1:

**Pull request merged**